### PR TITLE
scripts: error exit if aarch64 test is run on MSHV

### PR DESCRIPTION
**Pull request by russell-islam**
Signed-off-by: Muminul Islam <muislam@microsoft.com>

### CODE REVIEW

**Review by sboeuf — COMMENTED** [review]
@russell-islam looks good, but could you fix your commit message by replacing `aarch74` with `aarch64`?

**Comment by russell-islam** [comment]
> @russell-islam looks good, but could you fix your commit message by replacing `aarch74` with `aarch64`?

Meh!! Done.

**Review by sboeuf — APPROVED** [review]

File: scripts/run_integration_tests_aarch64.sh [hunk 1]
```diff
@@ -180,6 +180,12 @@ update_workloads() {
 
 process_common_args "$@"
 
+# aarch64 not supported for MSHV
+if [[ "$hypervisor" = "mshv" ]]; then
+    echo "Aarch64 is not supported in Microsoft Hypervisor"
+    exit 0
```
Comment (rbradford) [inline scripts/run_integration_tests_aarch64.sh:186]: I don't think it should exit 0. It's an error to try and combine msvh with aarch64.

**Review by rbradford — CHANGES_REQUESTED** [review]
I don't think it should ex

File: scripts/run_integration_tests_aarch64.sh (hunk 1 above)
Comment (russell-islam) [reply scripts/run_integration_tests_aarch64.sh]: Updated.

**Review by sboeuf — APPROVED** [review]

**Review by rbradford — APPROVED** [review]

**Pull request merged**